Makefile.vars.mk

```diff
@@ -16,6 +16,9 @@ CRD_FILE ?= k8up-crd.yaml
 CRD_ROOT_DIR ?= config/crd/apiextensions.k8s.io
 CRD_DOCS_REF_PATH ?= docs/modules/ROOT/pages/references/api-reference.adoc
 
+SAMPLES_ROOT_DIR ?= config/samples
+minio_sentinel = $(e2etest_dir)/minio_sentinel
```

Comment:
are you sure that the e2etest_dir gets created in a new, cleaned environment, without calling `make e2e-test` first? aka is make actually able to create the file without the dir? I'm not sure about that.

Reply:
The `kind-setup` target uses that folder, too: https://github.com/k8up-io/k8up/blob/master/e2e/kind.mk#L1

The `.kind` folder doesn't exist, so your first suggestion wasn't working.

Reply:
ah, I mixed that up. I thought kind stuff would go into `.kind` and e2e stuff into `.e2e`. I must have remembered it wrongly then. All good

Reply:
yeah I was a bit confused, too. The makefiles baesd on `go-bootstrap` use `.kind`.